docs/github/v2/06-remote-builder.md

```diff
@@ -0,0 +1,123 @@
+# Remote builder
+
+## What is it?
+
+GitHub Actions run on build machines provided by GitHub. You can see info about the specifications of the build machines [here]
+(https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners).
+
+For Unity projects the disc size is quite small. You may experience an error related to running out of disc space. You may also want to run the build on a server with more memory or CPU resources.
+
+**You can use Remote Builder to send a build from any supported git platform to any of the supported remote builder types.**
+
+## Key Features
+
+- **Run builds on a cloud provider like AWS.** _This is billed on demand and costs nothing except unless you store the builds (which is cheap) while no builds are running._
+
+✔️ **AWS** is supported.
+
+❌ **GCP** planned, Cloud Run needs to release support for NFS storage.
+
+❌ **Azure** planned, already supported.
+
+- **Run builds on a Kubernetes cluster.** _Usually the cluster needs to be up and running at all times at all times as starting up a cluster is slow. Using Google Cloud's Kubernetes Autopilot you can scale down to the free tier automatically while not in use._
+
+✔️ All Kubernetes providers that support persistent volumes are supported.
+
+### When to use GitHub self-hosted runner or Remote Builder
```

Comment:
```suggestion
### When to use GitHub self-hosted runner or remote builder?
```

Reply:
Gonna stop here correcting all `remote builder` and `Remote Builder`. Please pick one and replace it throughout :)